### ISSUE TITLE
Unable to install php 5.6

### ISSUE THREAD

**Issue by mickael83** [scoopinstaller/versions#21]
I am unable to install php 5.6. Here is the output when running  install command:

```shell
$ scoop install php56

Installing 'php56' (5.6.29).

The remote server returned an error: (404) Not Found.
```

**Comment by lukesampson** [comment]
This was fixed in 0d5181db7fe99f6f2e757b0a13ca85f656673b21 with an update to version 5.6.30.

**Comment by indrakaw** [comment]
Help! I'm having a same problem. It's newer php56.
```
> scoop install php56

Installing 'php56' (5.6.40) [64bit]


The remote server returned an error: (404) Not Found.
URL https://windows.php.net/downloads/releases/php-5.6.40-Win32-VC11-x64.zip is not valid
```
It seems all php56 and older gone to archives subdir.
https://windows.php.net/downloads/releases/archives/
